citra-launcher.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+report_error() {
+    read -r -d '|' MESSAGE <<EOF
+Unfortunately, Citra seems to have crashed.
+We kindly ask you to submit a bug report to <a href="https://github.com/flathub/org.citra_emu.citra/issues">https://github.com/flathub/org.citra_emu.citra/issues</a>.
+
+When submitting a bug report, please attach your <b>system information</b> and the <b>Citra log file</b>.
+You seem to be using ${XDG_SESSION_DESKTOP} ${DESKTOP_SESSION} (${XDG_SESSION_TYPE}):
+To obtain Citra log files, please see <a href="https://community.citra-emu.org/t/how-to-upload-the-log-file/296">this guide</a>.
+To obtain your system information, please install <tt>inxi</tt> and run <tt>inxi -v3</tt>. |
+EOF
+    zenity --warning --no-wrap --title "That's awkward ..." --text "$MESSAGE"
+}
+
+export QT_QPA_PLATFORM=xcb
```

Comment:
Probably not needed since we don’t allow the wayland socket into the flatpak. May be useful to leave this out so that the socket can be manually ensbled by devs for debugging.

Reply:
> Probably not needed since we don’t allow the wayland socket into the flatpak. May be useful to leave this out so that the socket can be manually enabled by devs for debugging.

Developers should not use this launcher for debugging purposes. You will be able to launch `citra-qt` from the development shell when launching via the developer mode.

Reply:
Makes sense I guess